db/organization.js

```diff
@@ -71,10 +71,11 @@ class OrganizationDbController {
     return { id, ...rest }
   }
 
-  async create ({ name, host, installationId, email, avatarUrl }) {
+  async create ({ name, host, installationId, description, email, avatarUrl }) {
```

Comment:
Is create ever called with description? I thought create was only called during GitHub auth/install, which doesn't have extra fields.

Reply:
correct this is actually just for tests but i didn't think it was harmful to include an optional field in this create method